pkgs/applications/misc/gImageReader/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, libuuid
+, sane-backends, podofo, libjpeg, djvulibre, libxmlxx3, libzip, tesseract
+, enchant, intltool, poppler, json-glib
+, ninja
+, python3
+
+# Gtk deps
+# upstream gImagereader supports Qt too
+, gtk3, gobjectIntrospection, wrapGAppsHook
+, gnome3, gtkspell3, gtkspellmm, cairomm
+}:
+
+let
+  variant = "gtk";
+  pythonEnv = python3.withPackages( ps: with ps;[ pygobject3 ] );
+in
+stdenv.mkDerivation rec {
+  name = "gImageReader-${version}";
+  version = "3.2.99";
+
+  src = fetchFromGitHub {
+    owner= "manisandro";
+    repo = "gImageReader";
+    rev = "v${version}";
+    sha256 = "19dbxq83j77lbvi10a8x0xxgw5hbsqyc852c196zzvmwk3km6pnc";
+  };
+
+  nativeBuildInputs = [
+    cmake ninja
+    intltool
+    pkgconfig
+  ]
+  ++ [ wrapGAppsHook gobjectIntrospection ];
```

Comment:
Why are you splitting the dependencies to two lists?

Reply:
I tried to isolate gtk-specific packages so that if someone wants to add Qt support, it can make some packages optional. 
As a follow up of your comment, rather than having 2 lists, I merged the 2 and added a comment "gtk specific"